changelog/issue-2662.md

```diff
@@ -0,0 +1,4 @@
+audience: deployers
+level: minor
```

Comment:
There's some advice on changelog levels [here](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md).  In this case, I suspect `silent` is the right choice, unless this is fixing an issue that a user has identified and will want to see fixed.

Reply:
Done